docs/model_developers/resource_sets.rst

```diff
@@ -10,6 +10,11 @@ Partial compiles
     Care should be taken to :ref:`implement this safely<partial-compiles-guidelines>`, and the user should be aware of
     :ref:`its limitations<partial-compiles-limitations>`.
 
+.. note::
+
+    For partial compiles through LSM, see :ref:`its documentation<partial_compile_lsm>` on how to manage resource sets for an
+    LSM service in addition the documentation below.
```

Comment:
This link references a blank page on the OSS documentation.

Reply:
I made it so the note is only visible on the iso docs.

Reply:
I think you forgot to push that change

Reply:
So I did. Pushed now.